### ISSUE TITLE
player name overlaps with the time in a multi room chat

### ISSUE THREAD

**Issue by mouzedrift** [ppy/osu#5769]
**Screenshots or videos showing encountered issue:** 
![osu_2019-08-19_07-04-05](https://user-images.githubusercontent.com/43358824/63240384-b7996880-c24f-11e9-9264-ea20e85360b7.png)

**osu!lazer version:** 2019.814.0

**Comment by smoogipoo** [comment]
Duplicate of #2819. Will be resolved when we have truncation support, but this is blocked on framework fixed from my side (text builder + text anchors).
